Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/storage/legacy/RiskLevelResultMigrator.kt

```diff
@@ -34,12 +33,12 @@ class RiskLevelResultMigrator @Inject constructor(
         }
     }
 
-    private fun lastCalculatedRiskLevel(): RiskLevel? {
+    private fun lastCalculatedRiskLevel(): RiskState? {
         val rawRiskLevel = prefs.getInt("preference_risk_level_score", -1)
         return if (rawRiskLevel != -1) mapRiskLevelConstant(rawRiskLevel) else null
     }
 
-    private fun lastSuccessfullyCalculatedRiskLevel(): RiskLevel? {
+    private fun lastSuccessfullyCalculatedRiskLevel(): RiskState? {
```

Comment:
rename to state

Reply:
In this case "level" is correct because it refers to the legacy data we persisted with "levels", see `MigrationRiskLevelConstants`.